packages/core/src/browser/frontend-application.ts

```diff
@@ -26,6 +26,8 @@ import { preventNavigation, parseCssTime, animationFrame } from './browser';
 import { CorePreferences } from './core-preferences';
 import { WindowService } from './window/window-service';
 
+import '@vscode/codicons/dist/codicon.css';
```

Comment:
Should we import in the `frontend-application-module.ts` instead?

https://github.com/eclipse-theia/theia/blob/2470421f97dfe030824c52c0df023a8c29dd8d69/packages/core/src/browser/frontend-application-module.ts#L17-L22

Reply:
Sure, makes sense.